docs/development/api.md

```diff
@@ -39,6 +39,7 @@ APIs with no authorisation:
 | `/api/logout` | GET | Logout that will always succeed |
 | `/api/oauth/return` | GET | OAuth return request |
 | `/api/oauth/:provider` | GET | Provides access and refresh tokens if authorised |
+| `/api/organisation` | GET | Provides github organisation hostname |
```

Comment:
Every other reference in this PR is for `/api/threatmodel/organization`

Reply:
Much appreciated, I have fixed the typo with a further commit